kebechet/managers/README.rst

```diff
@@ -36,5 +36,22 @@ If you wish to operate on repository source code, you can request to clone it:
 
 The last thing you need to do, is to register your manager to `REGISTERED_MANAGERS` constant (you can find it in `kebechet/managers/__init__.py` file) so Kebechet knows about your manager. Manager can be referenced by its name in lowercase (class name without the "manager" suffix).
 
+Overlays
+========
+
+Thoth allows users to specify overlays consisting of different hardware and software environments.  These environments
+are specified in a users .thoth.yaml file, files associated with a specific overlay are located in
+`<overlays-dir>/<environment-name>`.  Thoth will specify overlays to act on, if you create a manager which acts on a
```

Comment:
what do you mean by a manager acting on a specific overlay? Isn't the default behaviour, act on all overlays if `overlays_dir` key exists in .thoth.yaml?
In that case go over all runtime enviornment names in `runtime_environments`, check for overlays/runtime_environment_name and act on those opening PR with updated dependencies, if no match is found, open an issue saying you have this overlay but .thoth.yaml please be sure to use same name

If non`overlays_dir` key exists default to usual behaviour, so look for dependencies in the root and use the first runtime environment in the list in .thoth.yaml by default